### PR TITLE
Documentation: Move developer environment content to README

### LINKED ISSUE #576: Restructure Documentation
I have to restructure the documentation into 

- user documentation
- architectural documentation
- development documentation

TODO
- [x] Home into README
- [x] Remove Branch Protection 
- [x] Remove Coveralls 
- [x] Data into README (PlanPro and SUMO configuration generator)
- [x] Database management without identifier into README (make smaller)
- [x] Remove the definition of done
- [x] Remove Epics Milestones
- [x] Remove ORM
- [ ] ~Add a small Introduction about used technologies (e.g., peewee, marshmallow)~
- [x] A small introduction on how to use readme, wiki, website
- [x] Decorators for Database recreation into README
- [x] Introduction to the wiki about its structure
- [ ] Add an empty chapter to the wiki for missing components
- [x] Setup steps into README

### PR DESCRIPTION
**Pull request by antonykamp**
Fixes #576 

Restructures documentation by moving developer environment information from wiki to README.

## PR checklist

- [ ] Acceptance criteria fulfilled
- [x] Docs (code, wiki & diagrams) updated
- [ ] Dev-branch has been merged into local branch to resolve conflicts
- [ ] Tests and linter have passed AFTER local merge
- [ ] Another dev reviewed and approved

### CODE REVIEW

File: README.md
Comment (antonykamp) [inline README.md:1]: 👍

File: README.md [hunk 1]
```diff
@@ -1,20 +1,210 @@
...
 
-`dev` branch: [![CI](https://github.com/BP2022-AP1/bp2022-ap1/actions/workflows/python-app.yml/badge.svg?branch=dev)](https://github.com/BP2022-AP1/bp2022-ap1/actions/workflows/python-app.yml?query=branch%3Adev)
+A REST API for simulations and analysis of train traffic on the LEAG rail network. Create one of many component configurations, for example, for defining the interlocking, train schedules, and faults. Simulation configurations hold connections to the component configuration. You could add connections to component configurations. A run is the execution of the defined simulation.
 
-`main` branch: [![CI](https://github.com/BP2022-AP1/bp2022-ap1/actions/workflows/python-app.yml/badge.svg?branch=main)](https://github.com/BP2022-AP1/bp2022-ap1/actions/workflows/python-app.yml?query=branch%3Amain)
+**USED TECHNOLOGIES**
```
Comment (CR1337) [inline README.md:10]: Didn't we also use marshmallow?

Comment (CR1337) [inline README.md:10]: Should we list yaramo here?

Comment (antonykamp) [reply README.md:10]: Yes, but only to parse the body of the request. I'm not sure if we want to list every technology.

Comment (antonykamp) [reply README.md:10]: Good idea 👍

File: README.md [hunk 2]
```diff
@@ -1,20 +1,210 @@
...
+
+### PlanPro
+
+There is a ```planpro_helper.py``` in the ```data``` directory, which main method uses yaramo to get a PlanPro file from OSM with hardcoded coordinates. The result is saved into the ```planpro``` subdirectory with a name chosen in ```planpro_helper.py```.
+
+### SUMO configuration
+
+There is a ```sumo_config_helper.py``` in the ```data``` directory, which main method uses yaramo to get a sumo-config from a PlanPro file. The name can not be chosen, but is the one from the input PlanPro file. There is a directory inside the ```sumo``` subdirectory with that name. Inside there is the ```sumo-config```directory with the SUMO files.
+
+By running `bin/db` you can manage the `dev` database and the `test` database.
```
Comment (CR1337) [inline README.md]: There seems to be no `bin/db` anymore. Do you mean `db/local/db.py`?

File: README.md (hunk 1 above)
Comment (CR1337) [reply README.md:10]: Ok, you're right. We shouldn't add any small technology.

Comment (CR1337) [reply README.md:10]: I added it.

**Comment by antonykamp** [comment]
ready to merge? 🙃

**Comment by instant-sky** [comment]
There is one unresolved conversation, startet by @CR1337 , left. Do you know more about this, I have no knowledge of the db stuff 😬

File: README.md (hunk 2 above)
Comment (antonykamp) [reply README.md]: That is correct :D

**Comment by CR1337** [comment]
> There is one unresolved conversation, startet by @CR1337 , left. Do you know more about this, I have no knowledge of the db stuff grimacing

`bin/db` definitely doesn't exist anymore. I am not sure if `db/local/db.py` is used anymore. After implementing the db stuff at the beginning I didn't work with it. Does anyone use `db/local/db.py` at the moment? If not maybe just mention the poe commands for database management.

**Comment by antonykamp** [comment]
>  Does anyone use db/local/db.py at the moment?

The commands are integrated in `poe` commands 👍

**Review by LucasDerReisende — APPROVED** [review]
Looks good

**Pull request merged**